### PR TITLE
Increase test coverage in javascript/backend

### PR DESCRIPTION
**Pull request by dimkl**
## Type of change

- [x] 🐛 Bug fix

## Packages affected

- [x] `@clerk/clerk-js`
- [ ] `@clerk/clerk-react`
- [ ] `@clerk/nextjs`
- [ ] `@clerk/remix`
- [ ] `@clerk/types`
- [ ] `@clerk/themes`
- [ ] `@clerk/localizations`
- [ ] `@clerk/clerk-expo`
- [ ] `@clerk/backend`
- [ ] `@clerk/backend-core`
- [ ] `@clerk/clerk-sdk-node`
- [ ] `@clerk/edge`
- [ ] `@clerk/shared`
- [ ] `build/tooling/chore`

## Description

- [x] `npm test` runs as expected.
- [x] `npm run build` runs as expected.

This is the 1st PR (more upcoming) that will help us improve the test coverage of the `cleck-js` package. With this we have updated/added some tests that were not included in the running suite of `backend` package! 

**Notes:**
I have added 2 TODOs comments that i am still looking into how to resolve.

https://linear.app/clerk/issue/JS-90/increase-test-coverage-for-token-verification-and-interstitial-in

### CODE REVIEW

**Comment by SokratisVidros** [comment]
@dimkl Can you please rebase this one to start the review?

File: packages/backend/src/tokens/keys.test.ts [hunk 1]
```diff
@@ -1,31 +1,15 @@
...
-    'cQIDAQAB\n' +
-    '-----END PUBLIC KEY-----';
-  const expectedJWK = {
-    kid: 'local',
-    kty: 'RSA',
-    alg: 'RS256',
-    n: '8Z1oLQbaYkakUSIYRvjmOoeXMDFFjynGP2-gVy0mQJHYgVhgo34RsQgZoz7rSNm_EOL-l_mHTqQAhwaf9Ef8X5vsPX8vP3RNRRm3XYpbIGbOcANJaHihJZwnzG9zIGYF8ki-m55zftO7pkOoXDtIqCt-5nIUQjGJK5axFELrnWaz2qcR03A7rYKQc3F1gut2Ru1xfmiJVUlQe0tLevQO_FzfYpWu7-691q-ZRUGxWvGc0ays4ACa7JXElCIKXRv_yb3Vc1iry77HRAQ28J7Fqpj5Cb-sxfFI-Vhf1GB1bNeOLPR10nkSMJ74HB0heHi_SsM83JiGekv0CpZPCC8jcQ',
-    e: 'AQAB',
-  };
-
```
Comment (dimkl) [inline packages/backend/src/tokens/keys.test.ts:28]: @chanioxaris i moved the PEM JWT mocks from `tokens/keys` to the `tokens/fixtures` module where we already have some JWT mocks.

**Review by SokratisVidros — APPROVED** [review]
👏 👏

File: packages/backend/src/tokens/keys.test.ts (hunk 1 above)
Comment (chanioxaris) [reply packages/backend/src/tokens/keys.test.ts:28]: Awesome 💯

**Review by nikosdouvlis — COMMENTED** [review]
Very nice 💯

**Pull request merged**